### PR TITLE
fix: before_first_request deprecation

### PR DESCRIPTION
**Pull request by utnapischtim**

### CODE REVIEW

File: invenio_administration/finalize_app.py
Comment (slint) [inline invenio_administration/finalize_app.py:1]: Not a big fan of introducing a new filename convention for these type of functions. Could we maybe instead leave these functions `ext.py`?

**Review by slint — APPROVED** [review]
Overall looks clean and as you mentioned pretty simple (even for more complex cases like this one). I guess there's always a tiny bit of modifications needed in keeping context/variables on the extension object, but it looks clean.

File: invenio_administration/finalize_app.py
Comment (utnapischtim) [reply invenio_administration/finalize_app.py:1]: if i move all what it is now in `finalize_app.py` should i keep the function structure the same. so one function `def finalize_app` as the entrypoint within the `ext.py` file?

Comment (slint) [reply invenio_administration/finalize_app.py:1]: Yeah, I think splitting into multiple entrypoints doesn't achieve anything, so one big function calling smaller functions doing their thing, looks like the way to go. Then the entrypoint is just pointing to `invenio_administration.ext:finalize_app`.

**Pull request merged**